spec/loaders/ruby_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../../lib/forspell/loaders/ruby'
+
+RSpec.describe Forspell::Loaders::Ruby do
+
+  let(:path) { File.join __dir__, '..', 'fixtures', 'example_module.rb' }
+
+  it 'should load words from ruby comments in file' do
+    loader = described_class.new(file: path, text: nil)
+    words = loader.read
+    expect(words).not_to be_empty
+    expect(words.map(&:text)).to include(*%w[erraccessor erralias This in the have])
+  end
+end
```

Comment:
There are this responsibilities for loader:
1. Extract words from comments (and don't extract from code)
2. Make sure that words "coordinates" (file/line) are correct
3. Drop everything that should not be tested

(3) is probably easier to test on its own (add `BaseLoader` test, or extract again :)) to a small module, which you can just test with "this word → dropped; this word → preserved")
1 & 2 should be tested in loader's tests -- maybe with just one huge list of "it should extract those words with this coordinates".